packages/legacy/core/App/screens/PINEnter.tsx

```diff
@@ -354,7 +354,9 @@ const PINEnter: React.FC<PINEnterProps> = ({ setAuthenticated, usage = PINEntryU
     if (store.lockout.displayNotification) {
       return (
         <>
-          <Text style={style.helpText}>{t('PINEnter.LockedOut')}</Text>
+          <Text style={style.helpText}>
+            {t('PINEnter.LockedOut', { time: String(store.preferences.autoLockTime ?? 5) })}
```

Comment:
I think 5 by itself is a magic number. In another PR circle back and define it somewhere like `const defautlAutolockInMinutes = 5`.